CONTRIBUTING.md

```diff
@@ -2,10 +2,52 @@
 
 Thanks for your interest in Gotify!
 
-When contributing to gotify/\*, please first discuss the change (typos excluded) you wish to make via issue, chat
-or any other method with the owners of this repository. If you want to fix an already existing issue please write 
-so in a comment.
+First of all, please note that we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project. 
 
-Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
+If you have any questions you can join the chat on [#gotify:matrix.org](https://matrix.to/#/#gotify:matrix.org).
 
-If you have any questions you can join the chat on [#gotify:matrix.org](https://matrix.to/#/#gotify:matrix.org).
\ No newline at end of file
+## Where to Contribute
+
+- [gotify/server](https://github.com/gotify/server) - server implementation and WebUI code. Written in Golang and React Typescript.
+- [gotify/android](https://github.com/gotify/android) - android client implementation. Written in Java.
+- [gotify/plugin-template](https://github.com/gotify/plugin-template) - official gotify plugin template. Written in Golang.
+- [gotify/cli](https://github.com/gotify/cli) - official CLI client. Written in Golang.
+- [gotify/website](https://github.com/gotify/website) - contents for [gotify.net](https://gotify.net/). Written in Markdown.
+
+
+## Ways to Contribute
+
+### Document Refinements
+
+_Keywords: **Documentation**, **Writing**_
+
+Documents are residing in the [gotify/website](https://github.com/gotify/website) repository. Open an issue or PR and indicate the part of the document you are working on or the information you want to add to the document.
+
+### Feature Request and implementation
+
+_Keywords: **Features**, **Coding**_
+
+When proposing features to gotify/\*, please first discuss the change (typos excluded) you wish to make via issue, chat or any other method with the maintainers.
```

Comment:
Not needed because we have a own section for docu.
```suggestion
When proposing features to gotify/\*, please first discuss the change you wish to make via issue, chat or any other method with the maintainers.
```